Sources/ProjectSpec/SwiftPackage.swift

```diff
@@ -7,24 +7,38 @@ public struct SwiftPackage: Equatable {
 
     public typealias VersionRequirement = XCRemoteSwiftPackageReference.VersionRequirement
 
-    public let url: String
-    public let versionRequirement: VersionRequirement
+    public enum Kind: Equatable {
```

Comment:
We can just make the  whole `SwiftPackage` the enum, without a need for a `Kind`